pallets/asset/src/checkpoint/mod.rs

```diff
@@ -236,7 +236,16 @@ decl_module! {
             storage_migrate_on!(StorageVersion::get(), 2, {
                 // We're making it into a double map due to a bug, nuke storage.
                 use polymesh_primitives::migrate::kill_item;
-                kill_item(b"Checkpoint", b"Timestamps");
+                for item in &[
+                    b"TotalSupply" as &[_],
+                    b"Balance" as &[_],
+                    b"BalanceUpdates" as &[_],
+                    b"ScheduleRefCount" as &[_],
+                    b"SchedulePoints" as &[_],
+                    b"Timestamps" as &[_],
+                ] {
+                    kill_item(b"Checkpoint", item);
```

Comment:
We need to either migrate the data instead of nuking it or nuke other data like `CheckpointIdSequence` as well and start from a fresh slate.

I think we should be migrating (because I want to test the migration tests :))

Reply:
Decided to nuke all of it.